znai/src/main/java/com/twosigma/znai/extensions/cli/CliCommandIncludePlugin.java

```diff
@@ -85,9 +100,28 @@ public SearchText textForSearch() {
         return SearchScore.HIGH.text(command);
     }
 
+    private void validateSplitAfter(String command, List<String> splitParts) {
+        String[] commandParts = command.split(" ");
+
+        for (String splitPart : splitParts) {
+            if (Arrays.stream(commandParts).noneMatch(commandPart -> commandPart.equals(splitPart))) {
```

Comment:
You can just create `HashSet` just like `validateParamsToHighlight`.

Reply:
do you mean to create a set from `commandParts`?  I don't see how having `splitParts` as `Set` helps.

Reply:
Set<String> commandParts = new HashSet<>(Arrays.asList(command.split(" "));
    for (String splitPart : splitParts) {
         if (!commandParts.contains(splitPart) {
                throw new RuntimeException("split part \"" + splitPart + "\" is not present in commands.");            
         }
    }